packages/bierzo-wallet/src/logic/transactions/types/BwUnkownTransaction/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Address, BlockchainConnection, ConfirmedTransaction, SendTransaction } from '@iov/bcp';
+import * as React from 'react';
+
+import { BwParser, ParsedTx } from '../../types/BwParser';
+import UnkownTransactionHeader from './ui/UnknownTxHeader';
+import UnkownTransactionRow from './ui/UnknownTxRow';
+
+export interface BwUnknownProps extends ParsedTx {
+  readonly id: string;
+  readonly received: boolean;
```

Comment:
Most unknown transactions do not have a direction like `SendTransaction`s. `RegisterUsernameTx` for example just has

```
  readonly kind: "bns/register_username";
  readonly username: string;
  readonly targets: readonly ChainAddressPair[];
  readonly fee?: Fee;
```

 Do we need this field? Would be easier if we can get rid of it for now

Reply:
I believe I can remove it, added because it was useful for some render logic, but I will figure out something. Good point. Thanks.

Reply:
If you need an intermediate step, set it to `false` by default since almost all transactions we list there are signed by the user